bin/merge_md_files.rb

```
@@ -1,76 +1,149 @@
+        rescue => e
+          puts "  Error processing BP #{bp_file}: #{e.message}"
+        end
       end
     end
   end
 
-  puts "Markdown files have been combined into #{output_file}"
-  puts "Total files processed: #{sorted_files.length}"
+  puts "Generated: #{output_file}"
+  puts "Total sections: #{file_order.length}"
 end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `generate_combined_file` is too high. [<22, 58, 20> 65.18/23]

(Metrics/AbcSize)